docs/tutorials/podman_pasta_ipv6_with_snid_ipv4.md

```diff
@@ -0,0 +1,761 @@
+
+Of course this requires (but should be already in Place, since the Rootless Containers require it already) in `/etc/sysctl.conf`:
+```
+net.ipv4.ip_unprivileged_port_start=80
+net.ipv4.ping_group_range=0 2000000
+kernel.unprivileged_userns_clone=1
+
+net.ipv6.ip_nonlocal_bind=1
+```
+
+Furthermore concerning the User Account Setup (it is preferable to use a different User compared to the one running the Podman Containers, for the Reasons outlined above):
+```
+groupadd snid
+useradd --shell /usr/sbin/nologin -g snid --base-dir /opt/snid snid
+chown snid:snid /opt/snid/snid
+chmod 0770 /opt/snid/snid
+```
+
+`/etc/systemd/system/snid-routes.service`:
+```
+# To get SNID to work:
+# - ip route add local the_nat46_prefix/96 dev lo
+# - nmcli connection modify lo +ipv6.routes "64:ff9b:1::/96 dev lo" -> NOT WORKING
+# - Backend CIDR: 2001:db8:0000:0001:0000:0000:0001:0001/112 (2001:db8:0000:0001:0000:0000:0001:0000 ... 2001:db8:0000:0001:0000:0000:0001:ffff)
+#
+# Convert IPv4 Address to IPv6 Address Representation: 
+# - https://www.agwa.name/blog/post/using_sni_proxying_and_ipv6_to_share_port_443
+# - https://www.rfc-editor.org/rfc/rfc6052
+
+[Unit]
+Description=SNID Routes Setup Service
+
+[Service]
+#Type=oneshot
+User=root
+ExecStart=/bin/bash -c 'ip route replace local 64:ff9b:1::/96 dev lo'
+ExecStop=/bin/bash -c 'ip route del local 64:ff9b:1::/96 dev lo'
+RemainAfterExit=yes
+
+[Install]
+WantedBy=multi-user.target
+```
+
+For the Service Part, we just put a Condition that requires `snid-routes.service` to be Run before (`Requires`).
+
+`/etc/systemd/system/snid-server.service`:
+```
+# To get SNID to work:
+# - ip route add local the_nat46_prefix/96 dev lo
+# - nmcli connection modify lo +ipv6.routes "64:ff9b:1::/96 dev lo" -> NOT WORKING
+# - Backend CIDR: 2001:db8:0000:0001:0000:0000:0001:0001/112 (2001:db8:0000:0001:0000:0000:0001:0000 ... 2001:db8:0000:0001:0000:0000:0001:ffff)
+#
+# Convert IPv4 Address to IPv6 Address Representation: 
+# - https://www.agwa.name/blog/post/using_sni_proxying_and_ipv6_to_share_port_443
+# - https://www.rfc-editor.org/rfc/rfc6052
+
+[Unit]
+Description=SNID Server Service
+Requires=snid-routes.service
+
+[Service]
+User=snid
+Group=snid
+ExecStart=/bin/bash -c 'cd /opt/snid && ./snid -listen tcp:172.16.1.10:443 -mode nat46 -nat46-prefix 64:ff9b:1:: -backend-cidr 2001:db8:0000:0001:0000:0000:0001:0001/112'
+
+# Not currently used
+#ExecStop=/bin/bash -c 'cd /opt/snid'
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Reload Systemd, enable and Start the Services:
+```
+# Reload Systemd Daemon
+systemctl daemon-reload
+
+# Enable Services
+systemctl enable snid-routes.service
+systemctl enable snid-server.service
+
+# Check for potential Errors
+systemctl restart snid-server.service
+systemctl status snid-server.service
+journalctl -xeu snid-server.service
+
+# Check for potential Errors
+systemctl status snid-routes.service
+journalctl -xeu snid-routes.service
```

Comment:
Maybe change to (code blocks can more easily copied this way):

---

Enable Services:
```bash
systemctl enable --now snid-routes.service snid-server.service
```

Check for Errors:
```bash
systemctl status snid-routes.service
systemctl status snid-server.service
```

---

Don't know if this is necessary, this isn't a `systemd` tutorial. `status` should contain most.
```bash
journalctl -xeu snid-server.service
journalctl -xeu snid-routes.service
```

Reply:
Implemented as Suggested. Removed the `journalctl` part.